src/components/Home/WorksList/Project/Project.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import data from "../../../../data.json";
+import "./Project.css";
+
+export default function Project(props) {
+    let proj;
+    data.forEach((project) => {
+        if (project.title === props.location.pathname.slice(9)) {
+            return (proj = project);
+        }
+    });
+    let { year, title, img, description } = proj;
```

Comment:
1. you should not use the location and slice .. you can use the match prop that the router forwards to your component .. try `console.log(props.match)` you will see the params object that has the title .. use this one :)) 
1. do you need `forEach` here ? why not `find` ? `const { year, title, img, description } = data.find(....)`

Reply:
beautiful :+1: